src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -77,6 +77,7 @@ void UsgsAstroFrameSensorModel::reset() {
     m_referencePointXyz.x = 0;
     m_referencePointXyz.y = 0;
     m_referencePointXyz.z = 0;
+    m_pixelGroundSize = 1.0;
```

Comment:
Why are we setting this to 1.0 in what amounts to the constructor?  Would it be better to set it to 0.0?

Reply:
This value is used in division, so I defaulted it to 1 to avoid division by 0 problems.

Reply:
It's being used as a denominator later, so I imagine to avoid divide by zero errors. Probably should have a more explicit way of dealing with it.

Reply:
Usually if a variable is used when it was not set correctly, it is better to fail ungracefully then to blindly move forward with a bad calculation. The problem presents its self much earlier with a catastrophic failure (instantly) then with a wrong answer at the end (possible never). @Kelvinrr an I talked about this, is it time to consider how best to handle errors within usgs csm?